<revisionlist>
  <revision rev="1" vrev="1">
    <srcmd5>3beab932f35765020214caa698109ade</srcmd5>
    <version>3.1.0</version>
    <time>1558532354</time>
    <user>dimstar_suse</user>
    <comment>Fake data generator</comment>
    <requestid>700464</requestid>
  </revision>
  <revision rev="2" vrev="1">
    <srcmd5>d13f55a006d9294b91418180c2126509</srcmd5>
    <version>3.2.0</version>
    <time>1560971315</time>
    <user>dimstar_suse</user>
    <comment></comment>
    <requestid>710558</requestid>
  </revision>
  <revision rev="3" vrev="1">
    <srcmd5>f5f67b030b58323d14699b2d5d6d0da9</srcmd5>
    <version>3.3.0</version>
    <time>1568379511</time>
    <user>lnussel_factory</user>
    <comment>- Update to 3.3.0:
  * country() from the Address() provider now by default returns the country name of the current locale.
  * Separated Europe and Asia continents in Italian locale.
  * Removed duplicated names in the countries of et locale.</comment>
    <requestid>730116</requestid>
  </revision>
  <revision rev="4" vrev="2">
    <srcmd5>8304d1626c13a7d7d1dca72d1dfddf6c</srcmd5>
    <version>3.3.0</version>
    <time>1578606681</time>
    <user>dimstar_suse</user>
    <comment>- Skip test test_cpf_with_666_prefix as it fails with new
  mocker behaviour</comment>
    <requestid>762090</requestid>
  </revision>
  <revision rev="5" vrev="3">
    <srcmd5>b8ca4022b746ced5a63bbae030e5f5b5</srcmd5>
    <version>3.3.0</version>
    <time>1583750543</time>
    <user>dimstar_suse</user>
    <comment>py38 fix</comment>
    <requestid>782839</requestid>
  </revision>
</revisionlist>
